### DISCUSSION TITLE
Interface não suportada

### DISCUSSION THREAD

**Discussion by Ljordison** [Daniel09Fernandes/Component_OpenOffice#32] [Announcements]
Delphi 10.2 Tokyo está dando erro de interface não suportada

**Comment by Daniel09Fernandes** [comment]
Neste caso, ao adicionar o path deu erro adiciona na, chave, no registro do windows.

Aperta Win + R, digita regedit e clica em OK

No meu caso aqui (RAD Studio 2010) fica em:
HKEY_CURRENT_USER\Software\CodeGear\BDS\7.0\Library

e adiciona em "Search Path".

*en*

In this case, adding the path gave an error, add the key to the windows registry.

Open Win + R, type regedit and click OK

In my case here (RAD Studio 2010) is:
HKEY_CURRENT_USER\Software\CodeGear\BDS\7.0\Library

and add in "Search Path".
